graphdatascience/session/dedicated_sessions.py

```diff
@@ -83,7 +87,9 @@ def get_or_create(
 
         connection_url = session_details.bolt_connection_url()
         if session_details.status != "Ready":
-            wait_result = self._aura_api.wait_for_session_running(session_id)
+            if timeout == 0:
+                raise RuntimeError(f"Failed to get or create session `{session_name}`.")
```

Comment:
I dont see where this would be a helpful for the user.
Either improve the message or not not have this case